source/documentation/administration_guide/chap-Renewing_certificates.adoc

```diff
@@ -18,16 +18,51 @@ Do _NOT_ let certificates expire. If they expire, the environment becomes non-re
 . Once the host is in maintenance mode, click menu:Installation[Enroll Certificate].
 . Once enrollment is complete, click menu:Management[Activate].
 
-.Next steps
-You must also renew Engine certificates.
+[IMPORTANT]
+====
+You must also renew {engine-name} certificates.
+====
 
-. Run `engine-setup` from an SSH or console session on the {engine-appliance-name} system. The `engine-setup` script prompts you with some configuration questions.
+.Renewing certificates in a standalone {engine-name}
+. Log in as root via SSH or console session to the {engine-name} and run the following command:
 +
 ----
-# engine-setup
+# engine-setup --offline.
 ----
 +
+The `engine-setup` script prompts you with configuration questions.
+
 . Respond to the questions as appropriate or use an answers file.
++
+Be sure to answer the question about renewing certificates with "Y." This will renew the certificates without applying any updates.
++
+If you want to apply updates, follow the procedure in the Upgrade Guide.
+
+.Renewing certificates in a self-hosted engine
+
+. Log in as root via SSH or console session to the {engine-name} and a self-hosted engine hypervisor.
+
+. In the hypervisor, run the following command to place the environment in global maintenance mode:
++
+----
+# hosted-engine --set-maintenance –mode=global
+----
+. In the {engine-name}, run the following command to renew the certificates:
++
+----
+# engine-setup --offline.
+----
```

Comment:
Add the comments to this step:

`+`
Enter `Renew certificates? Y` to renew certificates without applying updates.

Reply:
I believe that this is the step in the engine-setup that they are referring to: 

"Found existing PKI files, but /etc/pki/ovirt-engine/ca.pem is missing. If you continue, Setup will overwrite existing PKI files with new ones, including /etc/pki/ovirt-engine/ca.pem. After Setup completes you must reinstall or re-enroll certificates for all your hosts.
 
  If /etc/pki/ovirt-engine/ca.pem was accidentally deleted, stop Setup, restore /etc/pki/ovirt-engine/ca.pem from backup (/etc/pki/ovirt-engine/certs/ca.der), and then run Setup again.
 
  Continue with Setup and overwrite existing PKI files? (**Yes**, No) [No]:

This is the only step in the engine-setup involving certificates where the answer is Yes or No, in other places, the answers are Automatic /Manual, or the organization name for the certificate

Reply:
This looks right. Where did you find it?

Reply:
So I would rewrite it like this:

(engine setup step)
+
Ensure that you enter "Continue with Setup and overwrite existing PKI files? (Yes, No) [No]:  Yes" to renew the {engine-name} certificates.

Reply:
AFAICT the question when expired(or about to expire) should be
                       'One or more of the certificates should be renewed, '
                        'because they expire soon, or include an invalid '
                        'expiry date, or they were created with validity '
                        'period longer than 398 days, or do not include the '
                        'subjectAltName extension, which can cause them to be '
                        'rejected by recent browsers and up to date hosts.\n'
                        'See {url} for more details.\n'
                        'Renew certificates? '

Reply:
and that 398 dates check is no longer valid, opened [#376](https://github.com/oVirt/ovirt-engine/pull/376)

Reply:
^^ https://github.com/oVirt/ovirt-engine/pull/376

I don't think we have to repeat the whole string, perhaps just that "renew certificates"

Reply:
@michalskrivanek  do we need to say Yes to the option "Continue with Setup and overwrite existing PKI files?
or do we only need to say Yes to the "Renew certificates" option

Reply:
that depends if someone removed some files or not. I would assmue that for a renewal we can document just the renew, not other recoveries.